_posts/en/newsletters/2021-08-11-newsletter.md

```diff
@@ -0,0 +1,168 @@
+and service providers can prepare for the upcoming activation of taproot
+at block height {{site.trb}}.*
+
+{% include specials/taproot/en/07-nonces.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [C-Lightning 0.10.1][] is a release
+  that contains a number of new features, several bug fixes,
+  and a few updates to developing protocols (including [dual
+  funding][topic dual funding] and [offers][topic offers]).
+
+- [Bitcoin Core 22.0rc2][bitcoin core 22.0] is a release candidate
+  for the next major version of this full node implementation and its
+  associated wallet and other software. Major changes in this new
+  version include support for [I2P][topic anonymity networks] connections,
+  removal of support for [version 2 Tor][topic anonymity networks] connections,
+  and enhanced support for hardware wallets.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #21528][] aims to improve the p2p propagation of full
+  node listening addresses.  Exposure to a diverse set of addresses is important
+  for nodes to be protected against network partitions such as [eclipse attacks][topic eclipse attacks].
+  When Bitcoin Core nodes receive an address message containing 10 or fewer
+  addresses, they forward it to 1 or 2 peers. This is the primary technique
+  used to self-advertise addresses, so sending to peers that would not relay
+  these addresses would effectively stop or "black hole" the propagation through
+  the network. Although propagation failures cannot be prevented in the malicious case, this
+  patch improves address propagation for the honest cases, such as for block-relay-only
+  connections or light clients.
+
+  This update identifies whether or not an inbound connection is a
+  candidate for forwarding addresses based on whether it has sent an address
+  related message over the connection, such as `addr`, `addrv2`, or `getaddr`. This behavior
+  change could be problematic if there is software on the network that
+  relies on receiving address messages but never initiates an address-related
+  message. Therefore, the author took care to circulate this proposed change
+  before it was merged, including posting it to the
+  [mailing list][addrRelay improvements] and researching
+  [other open source clients][addr client research] to confirm compatibility.
+
+- [LND #5484][] allows storing all data in a single external Etcd
```

Comment:
etcd is styled in lowercase:

```suggestion
- [LND #5484][] allows storing all data in a single external etcd
```

or we could make this a link:

```suggestion
- [LND #5484][] allows storing all data in a single external [etcd](https://etcd.io/)
```

Reply:
Happy to discuss this further to find a consensus opinion, but my own personal preference is that we treat program names as either proper nouns (i.e. capitalized) or as code (e.g. in backtics).  (Up to the author of a section which they want to use; they may even vary within the section as needed, e.g. with "LND" and "`lnd'".)

Reply:
It's not hugely important, but I'm not sure that we should do this. If a program has a widely used lower-case style, then how do we decide how to capitalize it? Why 'LND' instead of 'Lnd'? Why 'Etcd' instead of 'EtcD' (etcd stands for [etc distributed](https://dbdb.io/db/etcd))? Would we capitalize glibc as Glibc or GLibC?

Like I said, it's not really that important, so I'm happy to defer to your choice.

Reply:
In those cases, I think we do the same thing we already do for Bitcoin Core and C-Lightning: when referring to the project, we use the full name (Bitcoin Core, C-Lightning, Etc Distributed, GNU Lib C); when referring to a specific program or library, put it in backtics (`bitcoind`, `lightningd`, `etcd`, `glibc`).

My overall thought here is that I really like to make text as easy and fast to read as possible, and I think conventions like starting proper nouns with a capital letter contribute to that.